### PR TITLE
Ensure we build before publishing a package.

### PR DESCRIPTION
**Pull request by jerome-benoit**
Signed-off-by: Jérôme Benoit <jerome.benoit@sap.com>

## PR Checklist

- [x] Please add a description of your changes.
- [ ] We need your changes to come with unit tests in order to keep this project in quality and easy to maintain.
- [ ] Please add a link to the open issue or task that this pull request will resolve.
- [ ] Add a note to the changelog to indicate the change.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

If yes please indicate it and migration info into the CHANGELOG.md file.

---

### CODE REVIEW

**Comment by Shinigami92** [comment]
We could make use of `prepublishOnly` (https://docs.npmjs.com/cli/v6/using-npm/scripts) instead

**Comment by jerome-benoit** [comment]
Manually merged by @pioardi.

File: .github/workflows/npmpublish.yml [hunk 1]
```diff
@@ -36,9 +36,15 @@ jobs:
...
+
+      - name: Build
+        run: npm run build:prod
+
+      - name: Install Production
         run: npm ci --production
 
       - name: Publish
-        run: npm publish
+        run: npm publish --prepublishOnly
```
Comment (Shinigami92) [inline .github/workflows/npmpublish.yml:48]: This CLI flag doesn't exist :thinking:  
I think you misunderstood it
Cause this is already merged I will open a new PR to fix that

Comment (jerome-benoit) [reply .github/workflows/npmpublish.yml:48]: Of course it exists ;)
It even works.

Comment (Shinigami92) [reply .github/workflows/npmpublish.yml:48]: prepublishOnly is a lifecycle script -> https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-scripts

Please link a documentation about you are talking
I don't know the CLI flag you are talking about

**Pull request closed without merging**